templates/404.html

```diff
@@ -5,10 +5,20 @@
 {% block content %}
 <div class="p-strip">
   <div class="row">
-    <div class="col-6">
-      <h1>404: Page not found</h1>
-      <p class="p-notification__response">
-        {{ error }}
```

Comment:
Do we not care if there's a specific error any more?

Reply:
I don't think so. But I am probably wrong. 

What were the differentiating factors @tbille?

I understood there was a 'page not found' and a 'snap not found'. The latter use case happened because only stable snaps were surfaced and certain architecture dependencies. Both of those have been fixed. 

This new page _hopefully_ allows an onward journey for users to find what they need